### PR TITLE
Mac version of install script

### PR DESCRIPTION
**Pull request by nikolalsvk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,17 +72,17 @@ vagrant up
...
-alias bb="cd ~/boxbox && vagrant up && vagrant ssh -c 'tmux -2'"
+alias bb="cd ~/boxbox && vagrant up && vagrant ssh"
 alias bhalt="cd ~/boxbox && vagrant halt"
 ```
 
 ## License
 
 The MIT License (MIT)
-Copyright (c) 2016 Igor Sarcevic
+Copyright (c) 2016 Nikola Đuza
```
Comment (shiroyasha) [inline README.md]: Hah, nice :)

**Review by shiroyasha — CHANGES_REQUESTED** [review]
Looks good, but modify the readme xD

File: README.md (hunk 1 above)
Comment (nikolalsvk) [reply README.md]: I thought my name should be there since it's my fork, or am I seriously wrong?

Comment (shiroyasha) [reply README.md]: Not sure 🤔 maybe `Copyright (c) 2017 Igor Sarcevic, NIkola Djuza`

**Pull request merged**